src/com/owncloud/android/providers/FileContentProvider.java

```diff
@@ -839,12 +835,28 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 try {
                     db.execSQL("ALTER TABLE " + ProviderTableMeta.CAPABILITIES_TABLE_NAME +
                             " ADD COLUMN " + ProviderTableMeta.CAPABILITIES_SHARING_PUBLIC_MULTIPLE
-                            + " INTEGER " + " DEFAULT 0");
+                            + " INTEGER " + " DEFAULT -1");
```

Comment:
I updated it, since these are ```CapabilityBooleanType``` instances, that hold three values: FALSE (0), TRUE (1) and UNKNOWN (-1). UNKNOWN seems more appropriate as default.